open_instruct/grpo_vllm_thread_ray_gtrl.py

```diff
@@ -79,6 +79,8 @@
 )
 from transformers.integrations import HfDeepSpeedConfig
 from vllm import SamplingParams
+from peft import PeftModel, get_peft_model_state_dict
```

Comment:
oh re-adding peft model support? did you test this?

Reply:
No did not test it, but this seems standard in openrlhf / open reaosner codebase so I just added it back.

Reply:
sure, nw